src/services/message.service.ts

```diff
@@ -70,6 +81,19 @@ export class MessageService {
       try {
         // Runs the sent function, with the data pulled from the emoji key.
         lambda(args.args);
+
+        embedData.emojiData = embedData.emojiData.filter((e) => e.emoji != reaction.emoji.name);
+
+        if (embedData.emojiData.length > minEmotes) {
+          return;
+        }
+
+        if (options.cutoffMessage) {
+          msg.edit(options.cutoffMessage);
+          msg.suppressEmbeds(true);
```

Comment:
why?

Reply:
which part? You wanted a message to display once someone reacts to a bot reply, didn't you?

Reply:
why the `suppressEmbeds`

Reply:
To hide the embed it looks weird with it.

Reply:
Oh I think I see the problem you are thinking of.  I could probably check if a embed is in the cutoffMessage/ closingMessage then apply it if necessary.

Reply:
Well when you edit it, its no longer and embed.
And if you want to enforce no embeds, just make the data type a string

Reply:
Embeds are classified as an attachment so it does seem to stay over edits.

Reply:
Also I was talking about stopping it from hiding the embed if it just set and embed.

Reply:
> Embeds are classified as an attachment so it does seem to stay over edits.

That's really weird, i never knew that. That is fine then